.github/workflows/golangci.yml

```
@@ -18,10 +18,10 @@ jobs:
         with:
           go-version-file: go.mod
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v7
+        uses: golangci/golangci-lint-action@v8
         with:
           # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
-          version: v2.0.2
+          version: v2.1.6
```

Personally, I think the version can be set to `latest`, but what do you think?
```suggestion
          version: latest
```

---

Good suggestion! Will incorporate following PR!